wagtail/wagtailadmin/datetimepicker.py

```diff
@@ -1,8 +1,7 @@
-from __future__ import absolute_import, unicode_literals
-
-
 # Adapted from https://djangosnippets.org/snippets/10563/
 # original author bernd-wechner
+
+
```

Comment:
Why are these newlines added?

Reply:
`flake8` wasn't happy.

Reply:
OK then.
I didn’t remember it forced spaces between comments & functions, that’s not specified in PEP8 as far as I remember.

Reply:
"Surround top-level function and class definitions with two blank lines." Doesn't it fall under this rule? I deleted the imports in the top, so the comments may be regarded as "top-level function".

Reply:
This doesn’t make sense to separate a comment from the function/class it’s about.
This rule is not an absolute rule to follow: for example, we don’t put two newlines at the end of the file after a class or function definition.
PEP8 itself shows some comments just above function definitions in this part: https://www.python.org/dev/peps/pep-0008/#code-lay-out